src/sentry/models/grouptombstone.py

```diff
@@ -39,6 +40,8 @@ class GroupTombstone(Model):
         blank=True, null=True
     )
     actor_id = BoundedPositiveIntegerField(null=True)
+    times_seen = BoundedPositiveIntegerField(db_default=0, null=True)
```

Comment:
This should be `null = False` since I assume you want the `times_seen` to be set to 0 for all rows?